src/utils/api/qaCertificationsAPI.js

```diff
@@ -838,6 +838,36 @@ export const updateTestQualification = async (locId, testSumId, id, payload) =>
   }
 };
 
+export const getAppendixECorrelationSummaryRecords = async (locId, testSumId) => {
+  let url = `${config.services.qaCertification.uri}`;
+
+  // *** workspace section url (authenticated)
+  if (window.location.href.indexOf("workspace") > -1) {
+    url = `${url}/workspace`;
+  }
+
```

Comment:
Pls use the getApiUrl(path) function in the same file to get url for a path you specify. This way we minimize code duplicates.  You can use Test Qualification API call as an example.